autocloud/consumer.py

```diff
@@ -51,58 +51,49 @@ def consume(self, msg):
         images = []
         compose_db_update = False
         msg_body = msg['body']
+        status = msg_body['msg']['status']
 
-        if msg_body['msg']['status'] in STATUS_F:
+        if status in STATUS_F:
             location = msg_body['msg']['location']
             json_metadata = '{}/metadata/images.json'.format(location)
-
             resp = requests.get(json_metadata)
             compose_images_json = getattr(resp, 'json', False)
-
-            if compose_images_json:
-                compose_images_json = compose_images_json()
-
-                compose_images = compose_images_json['payload']['images']
-                compose_details = compose_images_json['payload']['compose']
-
-                compose_images = dict(
-                    (variant, compose_images[variant])
-                    for variant in VARIANTS_F
-                    if variant in compose_images
-                )
-
-                compose_id = compose_details['id']
-                rel = fedfind.release.get_release(cid=compose_id)
-                release = rel.release
-
-                compose_details.update({'release': release})
-
-                for variant in VARIANTS_F:
-
-                    if variant not in compose_images:
-                        continue
-
-                    for arch, payload in compose_images[variant].iteritems():
-                        for item in payload:
-                            relative_path = item['path']
-
-                            if not is_valid_image(relative_path):
-                                continue
-
-                            absolute_path = '{}/{}'.format(location,
-                                                           relative_path)
-
-                            item.update({
-                                'compose': compose_details,
-                                'absolute_path': absolute_path,
-                            })
-                            images.append(item)
-                            compose_db_update = True
+        else:
```

Comment:
We can just remove the `else` block here and set compose_images_json to None in the beginning